e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -52,4 +64,26 @@ describe('Upload Files', () => {
             });
         });
     });
+
+    it('MM-T12 Loading indicator when posting images', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+        cy.get('.post-image').should('be.visible');
+        cy.postMessage('{enter}');
+
+        // # Login as otherUser
+        cy.apiLogin(otherUser);
+
+        // # OtherUser creates posts in the channel
+        cy.postMessageAs({
+            sender: otherUser,
+            message: 'post1',
+            channelId: testChannel.id,
+        });
+
+        // * Verify image is not loading for each posts
+        cy.get('.image-loading__container').should('not.be.visible');
```

Comment:
Since the parent of the loading container is `image-container`, maybe this can be improved to,

```suggestion
        cy.get('.image-container').should('be.visible').find('.image-loading__container').should('not.exist');
```

But yeah, as @saturninoabril suggested, verify first that loading indicator shows up when posting several images.